### ISSUE TITLE
serve from https?

### ISSUE THREAD

**Issue by henryncal** [cartomatic/hgis#15]
Getting tiles from http is giving my security team (me) problems.   Any chance of your upgrading to serve from https?

Thanks!

**Comment by mika666** [comment]
yeah, our wgis wms does not get as much attention as we would like to...
anyway, if you're looking for a quick solution, you have some options to consider:
* create a simple proxy and output data with your own ssl cert bound
* use hgiswms[1..4].maphive.net - can't promise it will not change in the future though.

exposing services via https from cartomatic domain should happen one day, but I can't give you any dates at his stage.

btw, care to share your usage of our hgis wms?

**Comment by henryncal** [comment]
Dominik:

Thanks for your immediate response!

First:  I’m the guy who functioned as webmaster for mapywig.org some years ago.  You and I had conversations about incorporating some of the WIG collection in project,

    http://hgis.cartomatic.pl 

 I’m very glad the WIG maps were added.   This coverage has become a valuable resource for my own work.

- - -

What am I doing?  To answer your question:  Since about 2004, I’ve been tracing my maternal ancestors in Kletsk (Kleck) and Wysokie Litewskie (Vysokoye),  both now in Belarus.  I discovered no traces of them in the Old Country, even of those who emigrated to the U.S. before WWII , so I turned to documenting their communities on my web sites

    kletsk.org
    wysokie-litewskie.org
    brest-belarus.org

(Sorry, I’m trying to simplify a long, complex story as well as I can!)

Getting to the hgis wms… One thread of my work, with respect to Kletsk, led me to this story:

    https://www.youtube.com/watch?v=rJRsz1yyL2w

About a year before that video was made, I was contacted by a descendant of Aleksandra Cybulska, who was then approaching 100 years old.  Could I help the family locate the woman, Sonia Berkowicz,  whom Aleksandra and her late husband protected as a child, at the profound risk of their lives?   Yes,  I could. I knew Sonia from a completely different thread of my research on Kletsk.    Sonia was overjoyed to support the effort to award Aleksandra Sprawiedliwi wśród Narodów Świata recognition.    I am  proud beyond expression of having a small part in that.

Sonia wants me to tell her story on kletsk.org, and I want to do it.  Trouble is, Sonia was very young at the time, and badly traumatized, so her memories are not very coherent.  I’ve been struggling to understand what actually happened, and where, during the war.   I threw my only slightly-cooked working notes on this page:

    https://kletsk.org/KletskRegionSoniaHidingPlaces.i.html

I’m hoping that viewing the geographic information we have about Sonia’s ordeal against modern and WIG maps will suggest … something.

Note as you look at the two Leaflet-based maps on this page that the WIG layer fails on the latest Chrome, which punitively rejects mixed content, but such content — for the moment— is tolerated by the latest Firefox.  

Sonia was also sheltered by a family named Krepski.   A Krepski descendant found this page, learned that his late grandparents were true heroes —of which he had no clue— and  I contributed sketches of the Krepski farm. I’ll be adding those as another map layer.  Will this help explain Sonia’s story?  At first query, she didn’t remember anything about this episode.   Mapping might help -- the chances are very small — again, typical of this work.  

This is the most active and interesting case.  I also use the hgis wms layer in various places on my other sites, hoping that the juxtaposition of old and new helps other descendants visualize the homes of their ancestors.  At least they will see how these towns have grown — or shrunk, in some cases.

- - -

Thanks for offering quick solutions.   I’m really at the edge of my tech skills putting up simple Leaflet maps. Plus: my main interest here is history.  

I don’t know how to create a proxy and establish a ssl certificate for it.  I can probably teach myself, but I’d rather use the time drive up to Portland, Oregon, an hour away, to visit with Sonia, who is an amazingly warm and energetic person. And she might remember some new tidbits about her wartime experience.   (Another tiny chance of success.)

Using an alternative server sounds good, even if it is only temporary.   Maybe https on cartomatic.pl will arrive before hgiswms[1..4].maphive.net go away.  First test, one tile — success.  I’ll switch over and test with a strict CSP in the next few days.  Thanks!

Assuming all is well, should I leave the credit to cartomatic or switch it to maphive?

Thanks,

Henry

Henry Neugass
Salem, Oregon USA
